src/sentry/static/sentry/app/plugins/components/pluginIcon.tsx

```diff
@@ -88,10 +93,15 @@ const IntegrationIcon = styled('div')`
   background-image: url(${p => p.image});
 `;
 
-class PluginIcon extends React.Component {
+type Props = {
+  pluginId?: string;
+  size: number;
+};
+
+class PluginIcon extends React.Component<Props> {
```

Comment:
We could probably combine these into a single styled component, since really nothing special is happening here, and it would make the typing easier I think

```jsx
const PluginIcon = styled('div')<Props>`
  position: relative;
  height: ${p => p.size}px;
  width: ${p => p.size}px;
  border-radius: 2px;
  border: 0;
  /* this is so that there aren't gray boxes before load */
  background-color: #fff;
  display: inline-block;
  background-size: contain;
  background-position: center center;
  background-repeat: no-repeat;
  background-image: url(${({pluginId}) => (pluginId !== undefined && ICON_PATHS[pluginId]) || DEFAULT_ICON});
`
```